### PR TITLE
Populate the `AUTHORS.md` file

### PR DESCRIPTION
**Pull request by ianmkenney**
Populating the historic authors from MDAnalysis core.

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -20,6 +20,35 @@ The rules for this file:
...
+- Tone Bengtsen <tonebengsten@gmail.com>
+- Wouter Boomsma <wb@bio.ku.dk>
+- Richard Gowers <richardjgowers@gmail.com>
+- Oliver Beckstein <orbeckst@gmail.com>
+- Max Linke <max_linke@gmx.de>
+- Jonathan Barnoud <jonathan@barnoud.net>
+
+**2018**
+- Tyler Reddy <tyler.je.reddy@gmail.com>
+- Zeman <zeman@icp.uni-stuttgard.de>
```
Comment (orbeckst) [inline AUTHORS.md:34]: Johannes Zeman

File: AUTHORS.md [hunk 2]
```diff
@@ -20,6 +20,35 @@ The rules for this file:
...
+
+**2019**
+- Matthijs Tadema <M.J.Tadema@protonmail.com>
+
+**2020**
+- Rocco Meli <r.meli@bluemail.ch>
+- shfrz <@shfrz>
```
Comment (orbeckst) [inline AUTHORS.md:41]: Is this the same in the MDAnalysis AUTHOR file?

File: AUTHORS.md [hunk 3]
```diff
@@ -20,6 +20,35 @@ The rules for this file:
...
+**2021**
+- Philip Loche <ploche@physik.fu-berlin.de>
+- Paarth Thadani <paarth.thadani15@gmail.com>
+
+**2022**
+- Aya Alaa <a.m.a09@outlook.com>
+- Estefania Barreto-Ojeda <estefania.barretooje@ucalgary.ca>
 
 **2023**
+- chrispfae <@chrispfae>
```
Comment (orbeckst) [inline AUTHORS.md:53]: Is this the same in the MDAnalysis AUTHOR file?

(probably Chris Pfaendtner... but not sure)

**Review by orbeckst — CHANGES_REQUESTED** [review]
Can you please check if we can/should resolve the three names that only have handles? Thanks.

File: AUTHORS.md (hunk 1 above)
Comment (orbeckst) [reply AUTHORS.md:34]: See https://github.com/zemanj

File: AUTHORS.md [hunk 4]
```diff
@@ -20,6 +20,35 @@ The rules for this file:
...
+- Matthijs Tadema <M.J.Tadema@protonmail.com>
+
+**2020**
+- Rocco Meli <r.meli@bluemail.ch>
+- shfrz <@shfrz>
+- Irfan Alibay <@IAlibay>
+
+**2021**
+- Philip Loche <ploche@physik.fu-berlin.de>
+- Paarth Thadani <paarth.thadani15@gmail.com>
```
Comment (IAlibay) [inline AUTHORS.md:46]: Just having a quick look through there's a co-authored commit by @lilyminium on the related PR: https://github.com/MDAnalysis/mdanalysis/pull/3178/commits/bf0310e40edbcd8d40cbef482503a5a5a0f16052

Should she be added too?

Comment (ianmkenney) [reply AUTHORS.md:46]: oof, yes. I was going by the commit authors, let me make sure to add all co-authors

Comment (ianmkenney) [reply AUTHORS.md:46]: Looks like she was the only missing author, thanks for pointing that out!

File: AUTHORS.md (hunk 1 above)
Comment (ianmkenney) [reply AUTHORS.md]: Fixed

**Review by orbeckst — APPROVED** [review]

**Pull request merged**